src/components/Synthetics/GmList/GmList.tsx

```diff
@@ -480,24 +480,21 @@ function ApyTooltipContent() {
   return (
     <p className="text-white">
       <Trans>
-        <p>
+        <p className="mb-12">
           The APY is an estimate based on the fees collected for the past seven days, extrapolating the current
           borrowing fee. It excludes:
-          <br />
-          <br />
-          - price changes of the underlying token(s)
-          <br />
-          - traders' PnL, which is expected to be neutral in the long term
-          <br />
-          - funding fees, which are exchanged between traders
-          <br />
-          <br />
+        </p>
+        <ul className="mb-8 list-disc">
+          <li className="p-2"> price changes of the underlying token(s)</li>
```

Comment:
is it nesessary to have padding _and_ a leading space ?

Reply:
spaces removed